### PR TITLE
Feature/dm 2992/change log for gateway scan node retry

### PR DESCRIPTION
**Pull request by Oezge-Akin**

### CODE REVIEW

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-2992
+version: 10.20.42.0
+---
+Issue that caused ignore of OPC UA device gateway property that contains the count of retries for the node scan (`gateway.scanAddressSpace.retries`) is resolved.
+New OPC UA device gateway property is added that allows to configure pause length between retries (`gateway.scanAddressSpace.pauseMillisForRetry`) during address space scan.
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:18]:
```suggestion
A new OPC UA device gateway property has been added that allows to configure the pause length between retries (`gateway.scanAddressSpace.pauseMillisForRetry`) during the address space scan.
```

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+date:
+title: OPC UA device gateway node scan retry properties
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:3]:
```suggestion
title: Improved configuration for OPC UA device gateway node scan retries
```

Comment (BeateRixen) [reply content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:3]: Not sure here but the title should not just be a heading for the topic but a statement on what works better now

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md (hunk 1 above)
Comment (aaronraab) [reply content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:18]: Why don't we say pause time? Pause length sounds a bit strange

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-2992
+version: 10.20.42.0
+---
+Issue that caused ignore of OPC UA device gateway property that contains the count of retries for the node scan (`gateway.scanAddressSpace.retries`) is resolved.
```
[the remark this answers is not in the document]
Comment (aaronraab) [reply content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:17]: The issue of the OPC UA device gateway property that contains the count of retires for the node scan (`gateway.scanAddressSpace.retries`)is being ignored has been resolved.   -> This still does not say by whom this is being ignored. For what is this relevant?

[the remark this answers is not in the document]
Comment (Oezge-Akin) [reply content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:17]: we didn't want to give too much technical details but basically that property was not effective and opcua-device-gateway still used default retry value.

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md (hunk 1 above)
Comment (Oezge-Akin) [reply content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:18]: pause duration or pause time would be fine :)

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-2992
+version: 10.20.42.0
+---
+An issue has been resolved that the OPC UA device gateway property which contains the count of retries for the node scan (`gateway.scanAddressSpace.retries`) was ignored.
+A new OPC UA device gateway property has been added that allows to configure the pause length between retries (`gateway.scanAddressSpace.pauseMillisForRetry`) during the address space scan.
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:18]:
```suggestion
A new OPC UA device gateway property has been added that allows to configure the pause time between retries (`gateway.scanAddressSpace.pauseMillisForRetry`) during the address space scan.
```

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (aaronraab) [reply content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md]: but then the problem is not the property itself being ignored but the count -> 
The issue of the count of retires for the node scan (gateway.scanAddressSpace.retries) that is stored in the OPC UA device gateway property is being ignored has been resolved.

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-2992
+version: 10.20.42.0
+---
+An issue has been resolved that the OPC UA device gateway property which contains the count of retries for the node scan (`gateway.scanAddressSpace.retries`) was ignored.
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:17]:
```suggestion
The issue of the count of retries for the node scan (`gateway.scanAddressSpace.retries`) that is stored in the OPC UA device gateway property is being ignored has been resolved.
```

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Oezge-Akin) [reply content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md]: done

**Review by aaronraab — APPROVED** [review]

File: content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-2992
+version: 10.20.42.0
+---
+The issue of the count of retries for the node scan (`gateway.scanAddressSpace.retries`) that is stored in the OPC UA device gateway property is being ignored has been resolved.
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-42-0-opcua-gateway-scan-retry.md:17]:
```suggestion
The issue that the count of retries for the node scan (`gateway.scanAddressSpace.retries`) stored in the OPC UA device gateway property is being ignored has been resolved.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Oezge-Akin Can this one be merged?

**Pull request merged**